xnat_tools/dicom_export.py

```diff
@@ -15,7 +15,7 @@
 import os
 import sys
 import tempfile
-
+from datetime import datetime   
```

Comment:
Did you run black on this file?

Reply:
I actually think I forgot to run black at the very end.. I made some changes but didn't run. I need to learn more to automate that set up. But all run again after the changes that you suggested. 

Thanks for the thorough review!